app/views/notifier/review_orders.text.haml

```diff
@@ -0,0 +1,8 @@
+= text "body_intro", facility: @facility, smart: false
+
+- @accounts.each do |account|
```

Comment:
@jhanggi Not sure what you mean--do you want it to look different from this?
![screen shot 2017-02-21 at 11 12 09 am](https://cloud.githubusercontent.com/assets/4624197/23175946/b2d6833c-f826-11e6-9d0c-008e4fe4be39.png)